pixeltable/exec.py

```diff
@@ -658,8 +658,7 @@ def _exec_cohort(self, cohort: Cohort, rows: DataRowBatch) -> None:
                         _logger.debug(
                             f'Running NOS task {cohort.model_info.task}: '
                             f'batch_size={num_nos_batch_rows} target_res={target_res}')
-                        result = Env.get().nos_client.Run(
-                            task=cohort.model_info.task, model_name=cohort.model_info.name, **kwargs)
+                        result = Env.get().nos_client.Run(cohort.model_info.name, inputs=kwargs, method=cohort.model_info.default_method, shm=True)
```

Comment:
Do you have docs for ModelSpec? The use of default_method would might a comment here.

Reply:
Added comments there with links to docs: https://autonomi-ai.github.io/nos/docs/api/common/spec.html#nos.common.spec.FunctionSignature. Will be updating these shortly.